index.bs

```diff
@@ -11422,6 +11418,32 @@ and its value is an object called a <dfn id="dfn-legacy-factory-function" export
 which allows creation of objects that implement the interface.
 The characteristics of a legacy factory function are described in [[#legacy-factory-functions]].
 
+<div algorithm>
+    Some interface operations are defined directly over ECMAScript values,
```

Comment:
I don't think "interface operations" is right here, because both "interface" and "operation" have specific meanings in Web IDL which do not match the maplike/setlike functions.

I think an appropriate introduction would be something like

> Some ECMAScript methods defined in this section will perform an implementation check in their opening steps, to ensure they're being called on the correct kind of object and that the method is valid to call from the current context.

Reply:
Ah, right, that's a bad mixing of levels. Your suggested intro sounds great.